app/models/RelationshipRecordList.scala

```diff
@@ -14,15 +14,12 @@
  * limitations under the License.
  */
 
-package utils
+package models
 
-object DateUtils {
+import play.api.libs.json.Json
 
-  val DatePattern: String = "yyyyMMdd"
+case class RelationshipRecordList(relationships: Seq[RelationshipRecord], userRecord: Option[LoggedInUserInfo] = None)
 
-  def isFutureDate(date: String): Boolean = {
-    val format = new java.text.SimpleDateFormat(DatePattern)
-    val time = format.parse(date).getTime
-    time > System.currentTimeMillis()
-  }
-}
+object RelationshipRecordList {
+  implicit val formats = Json.format[RelationshipRecordList]
```

Comment:
This uses format because it is used within RelationshipRecordStatusWrapper which in itself uses format. The reason it uses format is because Json.toJson is called upon a domain object within a wiremock connector test. Ideally the test should be updated to use Json.

Reply:
future ticket?

Reply:
Yes. Eventually the RelationshipRecordStatusWrapper will be removed and the correct HTTP response will be evaluated together with implicit reads and some Wiremock testing. That has been set for a different ticket too.